src/DocumentDbTests/Reading/Linq/query_against_date_Tests.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Linq;
+using Marten.Testing.Documents;
+using Marten.Testing.Harness;
+using Xunit;
+
+namespace DocumentDbTests.Reading.Linq
```

Comment:
The namespace is wrong for this test. Should be `DocumentDbTests.Bugs`

Reply:
No worries folks, I take care of little housecleaning things on PRs like this.